### PR TITLE
Make org-mode restart on theme change.

### PR DESCRIPTION
**Pull request by mwillsey**
Solves problem where hidden org bullets would be the old background color.

### CODE REVIEW

**Comment by LemmingAvalanche** [comment]
> Solves problem where hidden org bullets would be the old background color.

Was this reported somewhere?

**Comment by mwillsey** [comment]
#4688 mentions it, but it was closed for another reason. The problem is still present.

**Comment by LemmingAvalanche** [comment]
If that issue is relevant to your commit, maybe you could reference it in your commit message?

**Comment by mwillsey** [comment]
Okay, I changed it.

File: layers/+spacemacs/spacemacs-org/packages.el [hunk 1]
```diff
@@ -59,6 +59,11 @@
                     (1 font-lock-comment-face prepend)
                     (2 font-lock-function-name-face)
                     (3 font-lock-comment-face prepend))))
+      ;; changing themes can leave artifacts in org mode, it should reload
+      (add-hook 'spacemacs-post-theme-change-hook
+                '(lambda ()
+                   (when (featurep 'org)
+                     (org-save-outline-visibility 'use-markers (org-mode-restart)))))
```
Comment (nixmaniack) [inline layers/+spacemacs/spacemacs-org/packages.el]: Would be better to extract that to a function and then add it to hook. Almost all the functions added to hooks in spacemacs are named functions (maybe "Unwritten Rule") which helps if someone wants to remove it from the hook.

**Comment by nixmaniack** [comment]
Why did you close? I think this is a nice fix for users who change themes a lot at runtime and use org-mode.

**Comment by mwillsey** [comment]
Sorry! I was doing some housekeeping and didn't realize this wasn't
merged. Feel free to take the changed and submit your own PR.

 - Max

**Pull request closed without merging**